Makefile

```diff
@@ -48,18 +42,53 @@ checks:
 		&& exit 1; \
 	fi
 
-.PHONY: build
-
 build:
 	@echo "------------------"
 	@echo "--> Build Kubera Auth Server Image"
 	@echo "------------------"
 	docker build . -f ./Dockerfile -t $(REPONAME)/$(IMGNAME):$(IMGTAG)
 
-.PHONE: push
+test:
+	@echo "------------------"
+	@echo "--> Running tests"
+	go test ./...
+
+coverage:
+	@echo "Avoid running this one in your dev setup"
+	@echo "------------------"
+	@echo "--> Running tests with coverage"
+	# TODO: Fix the code or set the envs via a for loop to help write tests
+	# @for i in "JWT_SECRET" "ADMIN_USERNAME" "ADMIN_PASSWORD" "CONFIGMAP_NAME" "DB_SERVER" "PORTAL_URL";    do  let $i="dummy" ; done
+	@ADMIN_USERNAME="a" ADMIN_PASSWORD="b" CONFIGMAP_NAME="c" DB_SERVER="d" PORTAL_URL="e" go test ./... -cover -coverprofile=coverage.txt -covermode=atomic
+	# Cleanup ENVs like a good citizen
+	@ADMIN_USERNAME="" ADMIN_PASSWORD="" CONFIGMAP_NAME="" DB_SERVER="" PORTAL_URL=""
+
+
+golint:
```

Comment:
Is this different from `golangci-lint run`??

Reply:
Yes, [golint](https://github.com/golang/lint) is a tool which is supported by [golangci-lint](https://golangci-lint.run/usage/linters/), we can consider removing this Makefile target as we use [revive, golint's cousin](https://revive.run/) in the CI.